packages/autorest.python/test/azure/legacy/Expected/AcceptanceTests/CustomPollerPager/custompollerpager/operations/_paging_operations.py

```diff
@@ -2026,6 +2026,7 @@ def _get_multiple_pages_lro_initial(
         )
         _request.url = self._client.format_url(_request.url)
 
+        _decompress = kwargs.pop("decompress", True)
```

Comment:
can we add `decompress` to documentation?

Reply:
Still discuss in https://teams.microsoft.com/l/message/19:meeting_MWYwYzEyMTEtYmQwZS00MTM5LWFhYjItMDU2N2M3ZDhmY2Vi@thread.v2/1720484049786?context=%7B%22contextType%22%3A%22chat%22%7D